### DISCUSSION TITLE
How to join the Slack community

### DISCUSSION THREAD

**Discussion by icrc-fdeniger** [apache/superset#22076] [General]
Hello, 
I try to join the slack community with the given link found in the doc ( https://apache-superset.slack.com/join/shared_invite/zt-16jvzmoi8-sI7jKWp~xc2zYRe~NqiY9Q#/shared-invite/error) but I get this message: "This link is no longer active"

Is there another link available ?
Thanks

**Comment by kamalkeshavani-aiinside** [comment]
@icrc-fdeniger You can try this url I got from the slack:
https://join.slack.com/t/apache-superset/shared_invite/zt-1jp6hjzrq-H0PlFtToyLWuPiJDuRWCNw
I will raise a PR to change the link in docs. Thanks for reporting.

**Reply by nandubatchu** [reply]
This one is also giving the same response: `This link is no longer active`

**Reply by kamalk-github** [reply]
@nandubatchu The Slack invitation url gets expired after 400 invitations. Please use below url.
https://join.slack.com/t/apache-superset/shared_invite/zt-1mh158vmv-y67mOpmxjJyOKmJTUmYGhw

**Reply by partha108paul** [reply]
PLease share a new link

**Reply by TentacioPro** [reply]
Is there any possibility to join the slack community
I'm really in need to sort things out on installation, understanding and implementing Apache Superset, and im hitting roadblocks here
Can anyone help?
@kamalk-github @icrc-fdeniger @kamalkeshavani-aiinside

**Comment by icrc-fdeniger** [comment]
thanks @kamalk-github I've been able to join

**Comment by XyNG4** [comment]
Could you share a new link again?

**Comment by DrJukka** [comment]
where would one find an active link ?

**Comment by partha108paul** [comment]
Please share a new link

**Comment by Doctor-Who** [comment]
Hello,
Link is no longer active.
More simple, what about create a Matrix Channel to avoid slack limitation and use opensource chat platform ?

**Comment by Fahlo** [comment]
Please share a new link

**Comment by MaheninaRandrianirina** [comment]
Share a new link please

**Comment by jakshi** [comment]
How to join SuperSet Slack? Link is no longer active.

**Comment by glaceforge** [comment]
How to join Superset slack community?

**Comment by Dens1002** [comment]
Could a new link be sent please?

**Reply by msyavuz** [reply]
http://bit.ly/join-superset-slack Here you go for future reference:
https://superset.apache.org/community
